atomicapp/constants.py

```diff
@@ -75,3 +75,6 @@
 PROVIDER_CONFIG_KEY = "providerconfig"
 PROVIDER_TLS_VERIFY_KEY = "providertlsverify"
 PROVIDER_CA_KEY = "providercafile"
+
+# Persistent Storage Formats
+PS_FORMAT = ["ReadWriteOnce", "ReadOnlyMany", "ReadWriteMany"]
```

Comment:
Ahhh i was against that since the name seemed a bit too long (80 character width limit). i'll change it.